karma.conf.js

```diff
@@ -74,6 +74,9 @@ module.exports = function(config) {
 							'imports?define=>false'
 						]
 					}
+				],
+				noParse: [
+					/\/sinon\.js/,
```

Comment:
Yeh it works with it being removed to. 

Had to upgrade Sinon to use the pre-release version so the faketimers would be included with the webpack fix. Fingers crossed it gets properly released soon.